src/main/scala/com/amazon/deequ/repository/AnalysisResultSerde.scala

```diff
@@ -302,6 +302,10 @@ private[deequ] object AnalyzerSerializer
         result.addProperty(ANALYZER_NAME_FIELD, "Histogram")
         result.addProperty(COLUMN_FIELD, histogram.column)
         result.addProperty("maxDetailBins", histogram.maxDetailBins)
+        if (histogram.aggregateFunction != Histogram.Count) {
```

Comment:
Can we add a comment here that:
- the reason we are excluding `Count` since it is the current default
- It was the only previously supported function in the past, and we exclude it for backwards compatibility

Reply:
Will add comments.